src/supertux/command_line_arguments.cpp

```diff
@@ -93,6 +93,7 @@ CommandLineArguments::print_help(const char* arg0) const
             << _(     "  --disable-sound              Disable sound effects") << "\n"
             << _(     "  --disable-music              Disable music") << "\n" << "\n"
             << _(     "Game Options:") << "\n"
+            << _(     "  --open-in-editor             Open given level in editor") << "\n"
```

Comment:
Use `--edit-level` to keep current style (e.g. `--disable-sound`, `--play-demo`).